MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
+import random
+
+for k in range(0,len(sonnetz)-1):
+	lines=sonnetz[k].split('\n') #splits into lines
+
+	def DictRhymes():
+		getRhyme(0, 2) #English sonnet rhyme scheme (lines that rhyme)
+		getRhyme(1, 3)
+		getRhyme(4, 6)
+		getRhyme(5, 7)
+		getRhyme(8,10)
+		getRhyme(9,11)
+
+
+	def getRhyme(num1, num2): #num1 and num2 inputs represent which lines are selected (rhyming lines)
+		lastword1=lines[num1].split(' ')[-1] #takes the last word of every other line
+		lastword2=lines[num2].split(' ')[-1]
+		if(sentiment(lastword1)[0]<=.25 and sentiment(lastword2)[0]<=.25):
+			if lastword1 in rhymeWords: #checks for existing list
+				rhymeWords[lastword1].append(''.join(lastword2)) #extends list if need be
+			else:
+				rhymeWords[''.join(lastword1)]=[''.join(lastword2)] #otherwise makes a new list
+			if lastword2 in rhymeWords:
+				rhymeWords[lastword2].append(''.join(lastword1))
+			else:
+				rhymeWords[lastword2]=[lastword1]
+
+	DictRhymes()
+
+	def getMarkov(): #backwards Markov b/c the end of the line is selected first
+		for i in range(0,12):
+			getMarkLine(i)
+
+	def getMarkLine(num1): #num1 is the line that is getting processed
+		splitLine=lines[num1].split(' ')
+		for i in range (1, len(splitLine)):
```

Comment:
This is a really cool way of writing a loop that goes backwards that I honestly have never thought of before....